website/sidebars.js

```diff
@@ -2,38 +2,43 @@ const sidebarSettings = {
   docs: [
     "docs/introduction",
     "docs/supported-data-platforms",
+    {
+      type: "category",
```

Comment:
havent checked this thoroughly but it looks right

Reply:
:pray: